xdocs/usermanual/properties_reference.xml

```diff
@@ -1510,6 +1510,18 @@ JMETER-SERVER</source>
     Set to zero to disable the size check and display the whole response.<br/>
     Defaults to: <code>10485760</code>
 </property>
+<property name="view.results.tree.max_line_size">
+    Maximum size (in characters) of the line in the displayed.<br/>
+    This property workarounds Bug 63620 since Swing hangs when displaying very long lines.<br/>
```

Comment:
*works around*? I am not sure, that workaround is a verb, but then, I am not a native speaker, either. :)